### PR TITLE
Use testinfo.desc

### PR DESCRIPTION
**Pull request by hegerj**
Instead of Makefile use testinfo.desc which has variables resolved.

### CODE REVIEW

File: tmt.spec [hunk 1]
```diff
@@ -79,7 +79,7 @@ Summary: Extra dependencies for the Test Management Tool
 Requires: tmt >= %{version}
 Requires: tmt-container >= %{version}
 Requires: tmt-testcloud >= %{version}
-Requires: vagrant python3-nitrate
+Requires: vagrant python3-nitrate Make
```
Comment (psss) [inline tmt.spec]: I guess that should be `make`?

    # dnf install -y Make
    Last metadata expiration check: 1:36:28 ago on Wed Mar 11 09:24:36 2020.
    No match for argument: Make
    * Maybe you meant: make
    Error: Unable to find a match: Make

Comment (hegerj) [reply tmt.spec]: yes, mistake during rebase
fixed e2e3953

**Comment by hegerj** [comment]
Fixes #131

**Comment by psss** [comment]
Merged in 3dff02a after a couple of adjustments in 878b19d. Thanks for the improvement!

**Pull request closed without merging**